tools/floodgate/js/utils.js

```diff
@@ -0,0 +1,17 @@
+export function getFloodgateUrl(url) {
+  if (!url) {
+    return undefined;
```

Comment:
Nit: Redundant. By default it returns `undefined`
```suggestion
    return;
```

Reply:
@meganthecoder Eslint throws  `consistent-return` error if I remove undefined. So I left it as-is. LMK if there is a better way to handle this.